main.js

```diff
@@ -52,13 +52,16 @@ ideaBoxArray.push(currentIdea)
 function displayIdeaCard() {
 cardContainer.innerHTML = ''
 for (var i = 0; i < ideaBoxArray.length; i++) {
+  /*
+    Indent all this HTML so it's easier to read
+    Also, you don't need two img tags for the favorite star use JS to replace the img src
+  */
   cardContainer.innerHTML += `
   <div class='idea-cards'>
   <div class='card-header-main'>
   <header class='card-header'>
   <button class='header-buttons'>
-  <img class="favorite-on hidden" src="assets/star-active.svg" alt="favorite on" data-type='favorite-button'>
-  <img class="favorite-off" src="./assets/star.svg" alt="favorite off" data-type='favorite-button'>
+  <img class="favorite-star off" src="./assets/star.svg" alt="favorite off" data-type='favorite-button'>
```

Comment:
You don't need two `img` tags for the favorite star use JS to replace the img src